pages/api/pointsOfInterest.ts

```diff
@@ -0,0 +1,9 @@
+export default async function handler(req: any, res: any){
+
+    const placesApi = `https://maps.googleapis.com/maps/api/place/textsearch/json?query=todo+in+paris&key=${process.env.NEXT_PUBLIC_GOOGLE_MAPS_API_KEY}`
+    const results = await fetch(placesApi)
+
+
+    await res.status(200).send(await results.json())
```

Comment:
I _could_ be wrong - but we may wanna send back `await results.json()` as `JSON.stringify(await results.json())` because I THINK it's more lightweight that way
But I could be making that up!

Reply:
I changed it, idk if it is either but if it's not then you're a liar :D